scripts/check_compliance.py

```diff
@@ -304,10 +347,7 @@ def run(self):
 
         if os.path.exists(self.DOCS_WARNING_FILE) and os.path.getsize(self.DOCS_WARNING_FILE) > 0:
             with open(self.DOCS_WARNING_FILE, "rb") as docs_warning:
-                log = docs_warning.read()
-
-                self.case.result = Error("Documentation Issues", "failure")
-                self.case.result._elem.text = log.decode('utf8')
+                self.add_failure(docs_warning.read().decode("utf-8"))
```

Comment:
Hey maybe this change from Error to Failure will provide some more transparency? see discussion at https://lists.zephyrproject.org/g/devel/topic/30039752

Found this interesting bit:
http://doc.scalatest.org/1.0/org/scalatest/junit/JUnitTestFailedError.html
"In JUnit 4, AssertionFailedError was made to extend java.lang.AssertionError, and the distinction between failures and errors was essentially dropped. However, some tools that integrate with JUnit carry on this distinction,..."

Reply:
Yeah, especially if the `Error` message is just dropped by default.